docs/sample_config.yaml

```diff
@@ -709,11 +709,11 @@ retention:
 #
 #allow_profile_lookup_over_federation: false
 
-# Uncomment to disable device display name lookup over federation. By default, the
-# Federation API allows other homeservers to obtain device display names of any user
-# on this homeserver. Defaults to 'true'.
+# Uncomment to allow device display name lookup over federation. By default, the
```

Comment:
Will you update the relevant section in the config manual (https://github.com/matrix-org/synapse/blob/aa5f5ede3331bf916f459137e9d0df3ac0743093/docs/usage/configuration/config_documentation.md?plain=1#L1038) to reflect the new reality? Thanks!

Reply:
Oh, yes! Thanks for the reminder, it'll take me some time to get used to doing that :)